code/src/languages/lark/utils/test.js

```diff
@@ -0,0 +1,10 @@
+import LarkClient from "../web/larkClient";
```

Comment:
It looks like you refactored doctest related code in this PR, right? That's great, the only thing I'd request is at least a one-line comment at the top of this file that says what it's used for, since "test.js" sounds like this might be a unit test file. I think it'd help to specify that it's used for doctests (assuming it is).